source/scripting/Entity.cs

```diff
@@ -37,37 +37,93 @@ public IntPtr MemoryAddress
 		}
 
 		/// <summary>
-		/// Gets or sets the health of this <see cref="Entity"/>.
+		/// Gets or sets the health of this <see cref="Entity"/> in int.
 		/// </summary>
 		/// <value>
-		/// The health from 0 - 100.
+		/// The health in int.
 		/// </value>
+		/// <remarks>if you need to get or set the value strictly, use <see cref="HealthFloat"/> instead.</remarks>
 		public int Health
 		{
 			get
 			{
-				return Function.Call<int>(Hash.GET_ENTITY_HEALTH, Handle) - 100;
+				return Function.Call<int>(Hash.GET_ENTITY_HEALTH, Handle);
```

Comment:
Not sure this is a good idea. Health below 100 does not seem to be used on normal entities, so it's really confusing.